### ISSUE TITLE
Some features available on wrong node types?

### ISSUE THREAD

**Issue by aubergine10** [CitiesSkylinesMods/TMPE#1231]
I've been working on this table for the wiki documentation, to show where tools/features are available, and found some really odd stuff, namely:

* Priority Signs - why are they available for transition nodes?
* Junction Restrictions: Enter Blocked Junction - why is this available at transition nodes?
* Junction Restrictions: U-Turn - why is this allowed on bend (corner) nodes?!

| What | Junction | Transition | Middle | Bend | Segment |
| ----- | :--------: | :---------: | :------: | :----: | :--------: |
| Lane Connector | ✔️ | ✔️ | ✔️ | ✔️ | ❌ |
| Lane Arrow        | ✔️ | ✔️ | ❌ | ❌ | ❌ |
| Priority Signs     | ✔️ | ✔️ | ❌ | ❌ | ❌ |
| Enter Blocked    | ✔️ | ✔️ | ❌ | ❌ | ❌ |
| Ped Crossing     | ✔️ | ✔️ | ❌ | ✔️ | ❌ |
| U-Turn (JR)       | ✔️ | ✔️ | ❌ | ✔️ | ❌ |
| Traffic Lights     | ✔️ | ✔️ | ❌ | ❌ | ❌ |
| Timed TL          | ✔️ | ✔️ | ❌ | ❌ | ❌ |
| Turn on Red      | ✔️ | ❌ | ❌ | ❌ | ❌ |
| Change Lanes   | ✔️ | ❌ | ❌ | ❌ | ❌ |
| Speed Limit      | ❌ | ❌ | ❌ | ❌ | ✔️ |
| Veh Restrict      | ❌ | ❌ | ❌ | ❌ | ✔️ |
| Park Restrict     | ❌ | ❌ | ❌ | ❌ | ✔️ |
| Bulk Applicator | ❌ | ❌ | ❌ | ❌ | ✔️ |

Notes:

* Turn on Red - only at junction nodes with traffic lights

**Comment by kianzarrin** [comment]
@aubergine10 by transition  node I assume you mean a node with two segments (probably because I used to use this term too)? In game nodes can have transition flag which is used to signify an intersection which is a transition from urban to highway (technically it also applies to nodes with roads of multiple levels and not necessarily highway). 

> Junction Restrictions: Enter Blocked Junction - why is this available at transition nodes?

maybe we want leave some space for the poor pedestrians to cross!

Bend nodes should be treated like middle node without any junction restrictions.

**Comment by aubergine10** [comment]
> by transition node I assume you mean a node with two segments

Yeah, but where they are different prefabs (eg. large road transitions to small road, or ground to elevated, etc). Like, why would I need _priority signs_ or _enter blocked junction_ features on such nodes?

**Comment by aubergine10** [comment]
Tagging: #365 , #345 , #854 , https://github.com/CitiesSkylinesMods/TMPE/issues/364#issuecomment-500140194 (and subsequent comments on that issue; I've not yet re-tested the issues mentioned there)
